src/objects/Error_Settings__c.object

```diff
@@ -70,6 +70,15 @@ To view these jobs, search for Scheduled Jobs in Salesforce Setup.</description>
         <type>TextArea</type>
     </fields>
     <fields>
+        <fullName>Sandbox_Jobs_Rescheduled__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Used to track when NPSP apex jobs have been deleted and rescheduled in a Sandbox. It should remain unchecked in all Production orgs</description>
```

Comment:
Sorry for the delay. If it's not customer facing, no need for me to review it.  And FWIW, it's clear to me anyway. :-)